.rubocop.yml

```diff
@@ -17,6 +17,7 @@ AllCops:
     - 'Gemfile.lock'
     - 'config.ru'
     - 'db/migrate/20220120101655_create_active_storage_tables.active_storage.rb'
```

Comment:
I would remove this line if you added `'db/**/*'`

Reply:
I changed